server/backend/api/app/models/model.py

```diff
@@ -748,3 +777,70 @@ class FamAppEnvironment(Base):
         },
     )
 
+
+class FamPrivilegeChangeType(Base):
+    __tablename__ = "fam_privilege_change_type"
+    __table_args__ = {"schema": "app_fam"}
+
+    privilege_change_type_code = Column(String(10), primary_key=True)
+    description = Column(String(100), nullable=False)
+    effective_date = Column(TIMESTAMP, server_default=func.now(), nullable=False)
+    expiry_date = Column(TIMESTAMP)
+    update_date = Column(TIMESTAMP)
+
+    privilege_change_audits = relationship(
+        "FamPrivilegeChangeAudit", back_populates="privilege_change_type"
+    )
+
+    def __repr__(self):
+        return f"<FamPrivilegeChangeType(privilege_change_type_code={self.privilege_change_type_code}, description={self.description})>"
+
+
+class FamPrivilegeChangeAudit(Base):
+    __tablename__ = "fam_privilege_change_audit"
+    __table_args__ = {"schema": "app_fam"}
+
+    # Columns
+    privilege_change_audit_id = Column(
+        BigInteger, Identity(start=1, increment=1), primary_key=True
+    )
+    application_id = Column(
+        BigInteger, ForeignKey("app_fam.fam_application.application_id"), nullable=False
+    )
+    change_date = Column(TIMESTAMP, nullable=False)
+    change_performer_user_details = Column(JSON, nullable=False)
+    change_performer_user_id = Column(
+        BigInteger, ForeignKey("app_fam.fam_user.user_id")
```

Comment:
should we specify nullable=True just to make it explicit? (I'm not sure what we've done in other places.)